doc/adr/0009-search-by-place.md

```diff
@@ -0,0 +1,19 @@
+# 9. search by place
+
+Date: 2017-11-03
+
+## Status
+
+Accepted
+
+## Context
+
+Pharmacies may only be located by postcode or outcode.
```

Comment:
I'd go with something along the lines of `There is a need to provide the ability for users to search by place name.` for the context.